sunnypilot/modeld_v2/modeld.py

```diff
@@ -44,15 +44,17 @@ class ModelState:
 
   def __init__(self, context: CLContext):
     try:
-      self.model_runner = TinygradRunner() if TICI else ONNXRunner()
+      self.model_runner = get_model_runner()
     except Exception as e:
       cloudlog.exception(f"Failed to initialize model runner: {str(e)}")
+      raise
 
     buffer_length = 5 if self.model_runner.is_20hz else 2
     self.frames = {'input_imgs': DrivingModelFrame(context, buffer_length), 'big_input_imgs': DrivingModelFrame(context, buffer_length)}
     self.prev_desire = np.zeros(ModelConstants.DESIRE_LEN, dtype=np.float32)
     if self.model_runner.is_20hz:
-      self.full_features_20Hz = np.zeros((ModelConstants.FULL_HISTORY_BUFFER_LEN, ModelConstants.FEATURE_LEN), dtype=np.float32)
+      # TODO-SP: for full features 20Hz, setting +1 to bugger len makes it work only for Filet o Fish. Will fail any other model
```

Comment:
It actually didn't fail with filet o fish locally lol. That doesn't mean this is right. We need to understand this better